### ISSUE TITLE
Videos repeatedly skip

### ISSUE THREAD

**Issue by edgan** [yuliskov/SmartTubeNext#1509]
A few versions ago of the beta videos started skipping during playback. Sometimes it is a few seconds every 30 seconds, other times it is nearly constant, and yet other times it won't do it for hours. Sometimes restarting the device or the app will fix it for a while, but not consistently. It is probably somewhat dependent on the video you are trying to watch.

I did try uninstalling 14.82, installing an older version, immediately updating, logging in to YouTube, and the issue came right back. So a fresh install doesn't help.

Device:
Nvidia Shield on the latest update

Versions:
14.82 skips
14.20 doesn't skip, I went back to this build because I already had the apk

**Comment by edgan** [comment]
Looks like I spoke too soon. 14.20 also seems to experience it. Which is weird, because this just started recently and that build is from months ago. This implies that the issue is created by some change by YouTube.

**Comment by edgan** [comment]
Turning off SponsorBlock fixed the skipping, and it hasn't come back over two days.

**Comment by edgan** [comment]
I also have experiencee it with 14.60 non-beta.

**Comment by edgan** [comment]
It seems my issue was my Shield controller. I disconnected it, and replaced it with a controller from a different Shield. I have not see this issue on the same Shield since.

It was purchased in January, but has already gone glitchy. I purchased it to replace the original controller that had also gone glitchy in a different way. From my research the controller is under a one year warranty. So I should be able to get it RMAed.
